### PR TITLE
Support KerasModel just on Keras backend

### PR DESCRIPTION
**Pull request by lvliang-intel**
## Type of Change

bug fix
API not changed

## Description

ILITV-2398: During studying this issue, I found current design would convert SavedModel to KerasModel on Tensorflow backend, it will cause the quantization fail. We should support KerasModel just on Keras backend.

## Expected Behavior & Potential Risk

TFHub saved model can be quantized successfully on stock tensorflow.

## How has this PR been tested?

Pre-CI and TFHub saved model test.

## Dependency Change?

None.

### CODE REVIEW

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4833/artifact/report.html) pass, but accuracy still not normal, please check may need to fix in another PR.

**Review by zehao-intel — APPROVED** [review]

**Review by chensuyue — APPROVED** [review]
